instaloader/instaloader.py

```diff
@@ -1220,6 +1257,7 @@ def download_profiles(self, profiles: Set[Profile],
                           highlights: bool = False,
                           stories: bool = False,
                           fast_update: bool = False,
+                          latest_stamps: Optional[LatestStamps] = None,
```

Comment:
We should add new arguments to the end of the parameter list, since users may invoke the function with positional arguments.